test/active_job/queue_adapters/amazon_sqs_adapter_test.rb

```diff
@@ -29,20 +29,37 @@ module QueueAdapters
         end
 
         it 'adds message_deduplication_id and default message_group_id if job does not override it' do
-          expect(client).to receive(:send_message)
-                              .with(
-                                {
-                                  queue_url: 'https://queue-url.fifo',
-                                  message_body: instance_of(String),
-                                  message_attributes: instance_of(Hash),
-                                  message_group_id: Aws::Rails::SqsActiveJob.config.message_group_id,
-                                  message_deduplication_id: instance_of(String)
-                                }
-                              )
+          expect(client).to receive(:send_message).with(
+            {
+              queue_url: 'https://queue-url.fifo',
+              message_body: instance_of(String),
+              message_attributes: instance_of(Hash),
+              message_group_id: Aws::Rails::SqsActiveJob.config.message_group_id,
+              message_deduplication_id: instance_of(String)
```

Comment:
Can we try to assert the default and the override deduplication_id below to ensure that the configuration is working as expected?

Reply:
I added some specs.